### PR TITLE
Add material icons

### PR DESCRIPTION
**Pull request by jmastr**

### CODE REVIEW

File: qml/Silica4v-play/ApplicationWindow.qml [hunk 1]
```diff
@@ -24,8 +24,18 @@ App {
...
 
+    FontLoader {
+        id: material
+        source: "qrc:/icons/MaterialIcons-Regular.ttf"
+    }
+
     Component.onCompleted: {
         // We need to access the dp() function from the Theme component
         BVApp.Theme.myApp = app
+        if (material.status == FontLoader.Ready) {
```
Comment (micuintus) [inline qml/Silica4v-play/ApplicationWindow.qml]: Can it happen that the font loader isn't ready on time? Is that an asynchronous business here?

File: qml/components-generic/VenueDetails.qml [hunk 2]
```diff
@@ -26,6 +26,7 @@ import QtQuick 2.2
 import Sailfish.Silica 1.0
 
 import BerlinVegan.components.generic 1.0 as BVApp
+import BerlinVegan.components.platform 1.0 as BVPlatform
```
Comment (micuintus) [inline qml/components-generic/VenueDetails.qml]: You can import both the generic as well as the platform specific components into the same `BVApp` namespace.

File: qml/Silica4v-play/ApplicationWindow.qml (hunk 1 above)
Comment (jmastr) [inline qml/Silica4v-play/ApplicationWindow.qml]: Naaa, probably not. I am going to remove the `if`.

File: qml/components-generic/VenueDetails.qml (hunk 2 above)
Comment (jmastr) [inline qml/components-generic/VenueDetails.qml]: Great, didn't know that. Will change.

File: qml/Silica4v-play/ApplicationWindow.qml (hunk 1 above)
Comment (micuintus) [inline qml/Silica4v-play/ApplicationWindow.qml]: You can also leave it in. I just wanted to make sure that we don't have a potential race here.

**Comment by micuintus** [comment]
Very nice that thing!
More and more, the app starts looking like a complete hing (on V-Play too).

**Pull request merged**